llumnix/instance_info.py

```diff
@@ -67,7 +68,7 @@ def __init__(self,
         self.finished_request_ids = None
 
         # For record statistics, assigned in backend engine.
-        self.instance_id = None
+        self.instance_id = instance_id
```

Comment:
In vLLM, instance_info might be first update by scheduler callback. There is no instance_id in there.